src/DssSpell.sol

```diff
@@ -60,159 +56,160 @@ contract DssSpellAction is DssAction {
+    address internal constant NAVIGATOR         = 0x11406a9CC2e37425F15f920F494A51133ac93072;
+    address internal constant PALC              = 0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A;
+
+    // BA Labs address
+    address internal constant RISK_WALLET_VEST  = 0x5d67d5B1fC7EF4bfF31967bE2D2d7b9323c1521c;
+    // SES address
+    address internal constant SES_WALLET        = 0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6;
+
+    // ---------- Launch Project Funds ----------
+    address internal constant LAUNCH_PROJECT_FUNDING = 0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F;
+
+    address internal immutable MCD_VEST_DAI             = DssExecLib.getChangelogAddress("MCD_VEST_DAI");
+    address internal immutable MCD_VEST_DAI_LEGACY      = DssExecLib.getChangelogAddress("MCD_VEST_DAI_LEGACY");
     address internal immutable MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
-
-    // ---------- Andromeda Legal Expenses ----------
-    address internal constant BLOCKTOWER_WALLET_2 = 0xc4dB894A11B1eACE4CDb794d0753A3cB7A633767;
+    GemAbstract internal immutable MKR                  = GemAbstract(DssExecLib.mkr());
+
+    // Function from https://github.com/makerdao/spells-goerli/blob/7d783931a6799fe8278e416b5ac60d4bb9c20047/archive/2022-11-14-DssSpell/Goerli-DssSpell.sol#L59
+    function _updateDoc(bytes32 ilk, string memory doc) internal {
+        ( , address pip, uint48 tau, ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+        require(pip != address(0), "DssSpell/unexisting-rwa-ilk");
+
+        // Init the RwaLiquidationOracle to reset the doc
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
+            ilk, // ilk to update
+            0,   // price ignored if init() has already been called
+            doc, // new legal document
+            tau  // old tau value
+        );
+    }
 
     // ---------- Trigger Spark Proxy Spell ----------
     // Spark Proxy: https://github.com/marsfoundation/sparklend/blob/d42587ba36523dcff24a4c827dc29ab71cd0808b/script/output/1/primary-sce-latest.json#L2
     address internal constant SPARK_PROXY = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
-    address internal constant SPARK_SPELL = 0x68a075249fA77173b8d1B92750c9920423997e2B;
 
-    // ---------- Payment Streams ----------
-    address internal immutable MCD_VEST_DAI          = DssExecLib.getChangelogAddress("MCD_VEST_DAI");
-    address internal immutable MCD_VEST_MKR_TREASURY = DssExecLib.getChangelogAddress("MCD_VEST_MKR_TREASURY");
-    GemAbstract internal immutable MKR               = GemAbstract(DssExecLib.mkr());
-    address internal constant ECOSYSTEM_FACILITATOR  = 0xFCa6e196c2ad557E64D9397e283C2AFe57344b75;
-
-    // ---------- Timestamps ----------
-    // 2023-12-01 00:00:00
-    uint256 internal constant DEC_01_2023 = 1701385200;
-    // 2024-11-30 23:59:59
-    uint256 internal constant NOV_30_2024 = 1733007599;
+    // ---------- Trigger Spark Proxy Spell ----------
```

Comment:
I think we don't need to separate section here, so please remove section header comment & newline